gitgud/levels/util.py

```diff
@@ -194,12 +192,40 @@ def test_level(level, test):
 
     return True
 
-
-class Level:
+class NamedList:
+    # Pass in list of tuples in format (reference, Object), where reference is of type str
+    def __init__(self, data=[]):
```

Comment:
I think it would be better to make the definition `def __init__(self, names, items):`, it just makes things more clear. We'll need to assert that they're the same length `assert len(names) == len(items)`, and then we can use `for name, item in zip(names, items)`